tests/testthat/helper-orderly.R

```diff
@@ -21,30 +21,6 @@ skip_on_windows <- function() {
   testthat::skip_on_os("windows")
 }
 
-skip_if_no_vault_server <- function() {
```

Comment:
`vaultr`'s server management bits are massively improved now - see https://vimc.github.io/vaultr/articles/packages.html for details. Red diff = best diff